### PR TITLE
Added devcontainer.

### PR DESCRIPTION
**Pull request by seriva**
Useful for the poor people using Windows. Similar to https://github.com/epiphany-platform/m-azure-basic-infrastructure/pull/17

### CODE REVIEW

File: examples/basic_flow_devcontainer/Makefile [hunk 1]
```diff
@@ -0,0 +1,59 @@
+ROOT_DIR := $(patsubst %/,%,$(dir $(abspath $(firstword $(MAKEFILE_LIST)))))
+
+AWSBI_IMAGE_NAME := epiphanyplatform/awsbi:0.0.1
+
+-include "$(ROOT_DIR)/vars.mk"
```
Comment (rafzei) [inline examples/basic_flow_devcontainer/Makefile:5]:
```suggestion
-include $(ROOT_DIR)/vars.mk
```

File: examples/basic_flow_devcontainer/Makefile [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	@cd $(M_WORKDIR) && $(MAKE) plan-destroy \
+		M_AWS_ACCESS_KEY=$(AWS_ACCESS_KEY) \
+		M_AWS_SECRET_KEY=$(AWS_SECRET_KEY)
+
+destroy: setup
+	@cd $(M_WORKDIR) && $(MAKE) destroy \
+		M_AWS_ACCESS_KEY=$(AWS_ACCESS_KEY) \
+		M_AWS_SECRET_KEY=$(AWS_SECRET_KEY)
+
+setup: $(M_SHARED)/shared/$(M_VMS_RSA_FILENAME)
```
Comment (rafzei) [inline examples/basic_flow_devcontainer/Makefile:55]:
```suggestion
setup: $(M_SHARED)/$(M_VMS_RSA_FILENAME)
```

File: examples/basic_flow_devcontainer/Makefile [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+$(M_SHARED)/shared/$(M_VMS_RSA_FILENAME):
```
Comment (rafzei) [inline examples/basic_flow_devcontainer/Makefile:57]:
```suggestion
$(M_SHARED)/$(M_VMS_RSA_FILENAME):
```

File: examples/basic_flow_devcontainer/Makefile [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+	$(error Usage: make (all/metadata/setup/init/plan/apply/output/all-destroy/destroy/destroy-plan/clean) )
+
+all: init plan apply
+all-destroy: destroy-plan destroy
+
+clean:
+	@rm -rf $(ROOT_DIR)/shared
+
+metadata: setup
+	@cd $(M_WORKDIR) && $(MAKE) metadata \
```
Comment (rafzei) [inline examples/basic_flow_devcontainer/Makefile:24]:
```suggestion
	@cd $(M_WORKDIR) && $(MAKE) metadata
```

**Review by rafzei — CHANGES_REQUESTED** [review]
To run it, I had to do the following changes.

**Review by rafzei — APPROVED** [review]
One more thing:
setup-env.sh file should have execution rights (only matter on linux env)

**Review by atsikham — APPROVED** [review]

**Pull request merged**